src/fsharp/FSharp.Core/option.fsi

```diff
@@ -170,6 +255,14 @@ module Option =
     /// <param name="option2">The second option.</param>
     /// <param name="option3">The third option.</param>
     ///
+    /// <remarks>
+    /// <code>
+    /// None |> Option.map3 (fun x y z -> x + y + z) None None = None
+    /// None |> Option.map3 (fun x y z -> x + y + z) None (Some 100) = None
+    /// None |> Option.map3 (fun x y z -> x + y + z) (Some 5) None = None
+    /// Some 10 |> Option.map3 (fun x y z -> x + y + z) None None = None
+    /// Some 10 |> Option.map3 (fun x y z -> x + y + z) (Some 5) (Some 100) = Some 115</code></remarks>
+    ///
```

Comment:
Use `|||>` see above

Reply:
Note this is the three-input piping operator

Reply:
Ack.